### ISSUE TITLE
Wrong curly brace is not closed error on function containing $'\n'

### ISSUE THREAD

**Issue by amiq-eda** [de-jcup/eclipse-bash-editor#105]
Error on function declaration (line 2)

```
Multiple markers at this line
- This curly brace is not closed. So function 'f1' is not valid.
- It seems this opening bracket is missing a closing one.
```
```
#!/bin/bash
f1() {
    IFS=$'\n'
}
```

**Comment by de-jcup** [comment]
Hello @amiq-eda 
I will fix it in next future. At the mean time (if the error bothers you ) you can change the marker level to WARNING (see https://github.com/de-jcup/eclipse-bash-editor/wiki/Validation) for more info.

**Comment by amiq-eda** [comment]
Thank you for fixing it, the error doesn't bother me so much so I'll wait for the next release.

**Comment by de-jcup** [comment]
@amiq-eda : You are wellcome, I just released 1.5.2 where the fix is done.

**Comment by amiq-eda** [comment]
I confirm that the fix is working. Thanks again for the fast turnaround!
